### DISCUSSION TITLE
Rasterio, Rioxarray, Whitebox Reprojection error

### DISCUSSION THREAD

**Discussion by Iamnyatichi** [giswqs/geog-312#150] [Q&A] [answered]
Hello sir, I have encountered the reprojection errors in the past, but I was able to pass through. I changed my PC and I am getting reprojection errors again from rasterio, to rioxarray, and whitebox
In rasterio lesson:
![image](https://github.com/user-attachments/assets/a37ab8c1-2251-4b6d-8163-a32a0da9e3b8)

In Rioxxary:
![image](https://github.com/user-attachments/assets/d7fa23b3-e4b3-4904-9176-80f852beeb62)

Whitebox:
![image](https://github.com/user-attachments/assets/03630489-28a6-494b-a50f-3d17c2da2129)

**Comment by giswqs** [comment, marked as answer]
See https://github.com/rasterio/rasterio/discussions/2721
